notebooks/5_CUR.ipynb

```diff
@@ -162,10 +162,10 @@
     "For the $j^{th}$ column, the importance score is given by\n",
     "\n",
     "\\begin{equation}\n",
-    "\\pi_j = \\sum_i^k (U_C)_{ij}^2,\n",
+    "\\pi_j = \\sum_i^k (U_C)_{ji}^2,\n",
```

Comment:
Aren't the columns of Uc still the eigenvectors? So we should still be summing over the rows, i.e., (U_C)_{ij}, correct? The inconsistency with the code comes in where the SVD returns the right singular vectors transposed, so that the eigenvectors are rows. So we should instead probably revert this change and rename the variable `U_C` to something like `U_CT` an add a code comment to this effect

Reply:
Here I use the formula mentioned in a [reference](https://doi.org/10.1063/1.5024611) for the fourth notebook. If we want to know the importance score of the j-th column in X, we sum the j-th coordinate of different columns (right singular vectors) in U_C, i.e., the first k coordinates of the j-th row in U_C.
![image](https://user-images.githubusercontent.com/56018055/98036480-6658a300-1e1a-11eb-92c9-5bd6cacaa274.png)

@rosecers sent me your new PCov-CUR paper. But its explanation of the summation process is not as detailed as the previous one. I don't quiet understand it.
![image](https://user-images.githubusercontent.com/56018055/98037777-62c61b80-1e1c-11eb-9429-53895f9fa680.png)

Reply:
In addition, the importance scoring formula I have seen in other papers is the same as in the first paper. They all sum the j-th coordinates of different columns in UC.
![image](https://user-images.githubusercontent.com/56018055/98038604-b2591700-1e1d-11eb-9f53-0139edb95f6d.png)
![image](https://user-images.githubusercontent.com/56018055/98038737-eaf8f080-1e1d-11eb-9be2-dfe60a4ef860.png)

Reply:
I once added a note to remind readers that sps.linalg.svds will return U_C^T containing right singular vectors as rows. So we have two different ways of summing before and after. But Rose said it would be unnecessary after a new PR. So I deleted it.
![image](https://user-images.githubusercontent.com/56018055/98040006-e6cdd280-1e1f-11eb-84b8-7d758fc48214.png)

![image](https://user-images.githubusercontent.com/56018055/98039765-8179e180-1e1f-11eb-99fa-57674caaf1d4.png)
**Note** that the Ct in U_Ct here refers to\ tilde {C}.

Reply:
Thank you for sending the other references and pointing this out -- your edit appears correct then. I'll double-check with @rosecers to make sure the PCov-CUR equation is made clearer.